config/prod.exs

```diff
@@ -11,7 +11,7 @@ import Config
 # before starting your production server.
 config :companies, CompaniesWeb.Endpoint,
   http: [:inet6, port: System.get_env("PORT") || 8080],
-  url: [scheme: "https", host: "elixir-companies.com"],
+  url: [scheme: "https", host: "bego-elixir-companies.fly.dev"],
```

Comment:
We need to change this back.

```suggestion
  url: [scheme: "https", host: "elixir-companies.com"],
```

Reply:
Ooops, sorry, missed it. Thank you for correcting it.

Reply:
No problem @begedin! Teamwork makes the dream work 🎉 Thanks for helping out. Need to make time in 2025 to give this site a good overhaul. Your thoughts and feedback are welcome!